src/eqProps.js

```diff
@@ -14,7 +14,7 @@ var equals = require('./equals');
  * @param {Object} obj1
  * @param {Object} obj2
  * @return {Boolean}
- *
+ * @deprecated since v0.18.0
```

Comment:
> The problem with providing lots of sugar is that users have less exposure to the underlying general-purpose functions, so knowledge is less transferable. Sugar is best when those reading the code understand the "macro expansion" that will take place.

Here I come and disagree again. I think it is up to each and everyone to gain an understanding of how the code works. 

The problem with *not* adding sugar to common functions is that I now have to write `R.eqBy(R.propEq('something'))` in every place where I used to have `R.propEq`

I could of course define `propEq` myself but this requires me to put it in some other module or a module wrapping ramda that I then have to import in all my projects.

What bothers me the most is that the decisions of what sugar should be included and what should not feels very arbitrary.